src/index.js

```diff
@@ -16,6 +15,19 @@ import schema from './options.json';
 import TaskRunner from './TaskRunner';
 
 const warningRegex = /\[.+:([0-9]+),([0-9]+)\]/;
+const internalCreateHash = (algorithm) => {
```

Comment:
I believe we want this negated. If we cant find crypto global package, require webpack create hash

Reply:
> I believe we want this negated. If we cant find crypto global package, require webpack create hash

If we negate this, it will not work because the crypto global package always exists, and Node >= 18 does not have the MD4 hash algorithm.